src/Compilers/CSharp/Portable/Parser/LanguageParser_InterpolatedString.cs

```diff
@@ -41,8 +39,19 @@ private ExpressionSyntax ParseInterpolatedStringToken()
             var originalToken = this.EatToken();
 
             var originalText = originalToken.ValueText; // this is actually the source text
-            var originalTextSpan = originalText.AsSpan();
```

Comment:
i ended up moving off of ROS<char> because the generics trick we use here can't work with them because we're not ona  runtime that allows-ref-structs in generics.